CHANGELOG.md

```diff
@@ -362,7 +361,6 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 **ENHANCEMENTS**
 - Add support for multiple users cluster environments by integrating with Active Directory (AD) domains managed via AWS Directory Service.
 - Install NVIDIA drivers and CUDA library for ARM.
-- Build Slurm with `slurmrestd` support.
```

Comment:
Why is this line being removed?

Reply:
Its weird that changelog is not consistent throughout the branches. 


This line was not in [release-3.7](https://github.com/aws/aws-parallelcluster-cookbook/blob/release-3.7/CHANGELOG.md) changelog and neither in [release-3.1](https://github.com/aws/aws-parallelcluster-cookbook/blob/release-3.1/CHANGELOG.md

Reply:
Let's keep this line please. We may consider fixing 3.7 later.

Reply:
I see the change in develop changelog was added in this https://github.com/aws/aws-parallelcluster-cookbook/pull/2446.

but not picked for release-3.7 branch

Reply:
Yep, I guess they didn't want to modify all the release branches.